git_toprepo.py

```diff
@@ -701,7 +716,9 @@ def get_config_loaders(
     def get_config_loader(self, name: str, config_dict: ConfigDict) -> ConfigLoader:
         loader_type = config_dict["type"][-1]
         if loader_type == "none":
-            config_loader = StaticContentConfigLoader("")
+            config_loader: Union[
+                StaticContentConfigLoader, LocalFileConfigLoader, GitRemoteConfigLoader
+            ] = StaticContentConfigLoader("")
```

Comment:
Does the following work?
```suggestion
        loader_type = config_dict["type"][-1]
        config_loader: Union[
            StaticContentConfigLoader, LocalFileConfigLoader, GitRemoteConfigLoader
        ]
        if loader_type == "none":
            config_loader = StaticContentConfigLoader("")
```

Reply:
Yes it seems so! I did not know you could declare variables like that :)